ui/components/topicPage.tsx

```diff
@@ -252,10 +258,18 @@ export function TopicDashboard({ topic }: { topic: Topic }) {
                     {row.isr.length}
                   </Td>
                 );
-              case "offset":
+              case "offsets":
                 return (
                   <Td key={key} dataLabel={"Offset"}>
-                    {row.offset.offset}
+                    {row.offsets?.earliest?.offset}
+                    /
+                    {row.offsets?.latest?.offset}
+                  </Td>
+                );
+              case "recordCount":
+                return (
+                  <Td key={key} dataLabel={"Record Count"}>
+                    { (row.offsets?.latest?.offset ?? 0) - (row.offsets?.earliest?.offset ?? 0) }
```

Comment:
I'm not sure if this is the ideal syntax to handle optional values, but it seems to work properly.